.renovaterc.json5

```diff
@@ -0,0 +1,42 @@
+{
+  assignees: [
+    "kevinmatthes",
+  ],
+  automerge: false,
+  commitMessageAction: "update",
+  commitMessageLowerCase: "never",
+  commitMessagePrefix: "⬆️",
+  dependencyDashboard: true,
+  dependencyDashboardAutoclose: true,
+  dependencyDashboardLabels: [
+    "dependencies",
+  ],
+  labels: [
+    "dependencies",
+  ],
+  lockFileMaintenance: {
+    commitMessageAction: "maintain Cargo.lock",
+    commitMessagePrefix: "🤖",
+    enabled: true,
+    schedule: "0 0 * * MON",
+  },
```

Comment:
Every Monday at UTC midnight, update the lock file (Cargo.lock) and commit the changes as "🤖 maintain Cargo.lock".

Reply:
Let start without the lockfile maintenance please. I'd like to see how it behaves without it.
```suggestion
```

Reply:
All in all good changes. Thank you @kevinmatthes.

Reply:
I just turned it off. :wrench: :heavy_check_mark:

It suffices to set the boolean switch `enabled` to `true` in order to activate this feature.  I left the configuration unchanged and only edited the line with `enabled`.